db/schema.rb

```diff
@@ -529,6 +530,31 @@
     t.index ["key"], name: "index_solid_queue_semaphores_on_key", unique: true
   end
 
+  create_table "statement_adjustments", force: :cascade do |t|
+    t.bigint "statement_id", null: false
+    t.uuid "api_id", default: -> { "gen_random_uuid()" }, null: false
+    t.string "payment_type", null: false
+    t.decimal "amount", default: "0.0", null: false
```

Comment:
Out of interest, why would we want a default amount? Is an adjustment of `0.0` actually an adjustment?

Reply:
Good point, we wouldn't (and we prevent it explicitly in the validation). I've just copied this from ECF without thinking 🙂 I'll fixup a change